R/pre-msa-tree.R

```diff
@@ -104,6 +112,25 @@ addLeaves2Alignment <- function(aln_file = "",
     lin_file = "data/rawdata_tsv/all_semiclean.txt", # !! finally change to all_clean.txt!!
     # lin_file="data/rawdata_tsv/PspA.txt",
     reduced = FALSE) {
+    
+    #Check if the alignment file is provided and exists
+    if (nchar(aln_file) == 0) {
+        abort("Error: Alignment file path must be provided.")
+    }
+    
+    if (!file.exists(aln_file)) {
```

Comment:
Should we remove extra spaces or use paste0 if we're introducing spaces? [or do the tests work out OK?]